pages/wishlist/wishlist.html

```diff
@@ -0,0 +1,291 @@
+                         
+                <div class="rating sm">
+                  <i class="fas fa-star"></i>
+                  <i class="fas fa-star"></i>
+                  <i class="fas fa-star"></i>
+                  <i class="fas fa-star"></i>
+                  <i class="far fa-star"></i>
+                  <span class="rating-info">(4)</span>
+              </div>
+                         
+              </div>
+       
+              <footer class="card-actions">
+                <button class="btn btn-primary card-btn-1">Move to Cart</button>
+                <button class="btn btn-secondary card-btn-2">Add to Cart</button>
+              </footer>
+       
+            </section>
+          </div>
+
+
+        <!-- card here-->
+
+          <!-- card here-->
+
+          <div class="card single-btn card-ecom card-w-dismiss">
+            <header class="card-image">
+              <img src="/images/barbell.webp" alt="product image" />
+            </header>
+       
+            <section class="card-body">
+              <div class="card-badge card-dismiss">
+                <div class="icon-container badge-container">
+                  <i class="fas fa-times icon"></i>
+                </div>
+              </div>
+       
+              <div class="card-badge card-like">
+                <div class="icon-container badge-container">
+                  <i class="fas fa-heart icon"></i>
+                </div>
+              </div>
+       
+              <div class="card-badge left-badge">
+                <div class="badge-container tag tag-danger">
+                  <span class="icon-badge">50% Off</span>
+                </div>
+              </div>
+       
+              <div class="card-overlay">Out of Stock</div>
+       
+              <div class="card-text-content">
+                <h1 class="card-title">Home Workout</h1>
+       
+                <p class="flexible-txt subtitle">Upto 50% Off.</p>
+       
+                <div class="ecom-qty">
+                  Quantity: <input type="number" min="1" max="50" />
+                </div>
+       
+                <p class="flexible-txt details">
+                <span class="strike-text gray-text">$1599</span> $749</p>
+                         
+                <div class="rating sm">
+                  <i class="fas fa-star"></i>
+                  <i class="fas fa-star"></i>
+                  <i class="fas fa-star"></i>
+                  <i class="fas fa-star"></i>
+                  <i class="far fa-star"></i>
+                  <span class="rating-info">(4)</span>
+              </div>
+                         
+              </div>
+       
+              <footer class="card-actions">
+                <button class="btn btn-primary card-btn-1">Move to Cart</button>
+                <button class="btn btn-secondary card-btn-2">Add to Cart</button>
+              </footer>
+       
+            </section>
+          </div>
+
+
+        <!-- card here-->
```

Comment:
you can write ```<!-- x card here x-->``` so it becomes easier to understand that card part has ended here.

Reply:
Thanks a lot, for your suggestion, I'll make the necessary changes asap
!